packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -69,7 +70,14 @@ const appendLogoutParam = (link: string): URL => {
 };
 
 const logout = async (logoutUrl: string, token: string): Promise<void> => {
-  await Promise.all([dispatchLogout(), logoutSession(logoutUrl, token)]); // temporary
+  await Promise.all([
+    dispatchLogout(),
+    submitKratosFlow({
+      action: logoutUrl,
+      params: { csrf_token: token },
+      method: 'GET',
+    }),
+  ]); // temporary
```

Comment:
The API logout already handles Kratos logout, no need to extra call it.

Reply:
Awesome. We should remove this then. Thank you for the confirmation.